src/Symfony/Component/VarDumper/Resources/functions/dump.php

```diff
@@ -31,6 +31,26 @@ function dump($var, ...$moreVars)
     }
 }
 
+if (!function_exists('dumpif')) {
+    /**
+     * @author Tales Santos <tales.augusto.santos@gmail.com>
+     */
+    function dumpif($var, ...$moreVars)
+    {
+        $condition = array_pop($moreVars);
+
+        if (!is_callable($condition)) {
+            throw new \InvalidArgumentException('You should provide a condition in order to dump $var');
+        }
+
+        if (true !== call_user_func($condition)) {
```

Comment:
I would pass the vars into the callable's parameters

Reply:
Good idea, but looks like this PR will not be accepted.